app/controllers/api/v1x0/portfolio_items_controller.rb

```diff
@@ -37,8 +37,10 @@ def show
       end
 
       def destroy
-        PortfolioItem.find(params.require(:id)).discard
-        head :no_content
+        item = PortfolioItem.find(params.require(:id))
+        svc = Catalog::SoftDelete.new(item)
```

Comment:
Nitpicking here, but I think the `.process` should be on this line, it just seems odd to push it into the `render` call instead of calling `.process` like we do with most of the services and then do the data access in the `render`.

Also nitpicky, but can we change the name from `svc` to something like `soft_delete` or something else that's a lot more easily searchable later?